tests/fixtures/rules/missing-client-directive-value/valid/client-only-value.astro

```diff
@@ -0,0 +1,8 @@
+---
+import SomeSvelteComponent from "../SomeSvelteComponent.svelte"
+let string = "svelte"
+---
+
+<!-- prettier-ignore-start -->
+<SomeSvelteComponent client:only={string} />
+<!-- prettier-ignore-end -->
```

Comment:
Could you please add a test case to check that there are no false positives with other directives?
For example, `client:idle`.

Reply:
Yes, good point.